addons/danger/functions/fnc_leaderManoeuvre.sqf

```diff
@@ -42,14 +42,35 @@ _pos pushBack (_target call CBA_fnc_getPos);
 // adds movement order
 (group _unit) move (_pos select 0);
 
+// debug
+[_unit, format ["units %1 positions %2", count _units, count _pos]] call FUNC(dotMarker);
+systemchat format ["units %1 positions %2", count _units, count _pos];
+
+{
+    _sign = "Sign_Arrow_Yellow_F" createVehicle getpos _x;
+    [_x,"units","colorYellow"] call FUNC(dotMarker);
+} foreach _units;
+
+{
+    _sign = "Sign_Arrow_Blue_F" createVehicle _x;
+    [_x,"positions","colorBlue"] call FUNC(dotMarker);
+} foreach _pos;
+
 // manoeuvre CQB // within close combat - nkenny
 if (_unit distance2D (_pos select 0) < GVAR(CQB_range)) exitWith {
+
+    // mass strenght one spot
+    _pos = selectRandom _pos;
+
     {
         _x forceSpeed 2;
-        _x doMove selectRandom _pos;
+        _x doMove _pos;
 
         // force movement
-        if !(_x call FUNC(indoor)) then {_x playActionNow selectRandom ["FastF", "FastF", "FastLF", "FastRF"];};
+        if !(_x call FUNC(indoor)) then {[_x, ["FastF", "FastF", "FastLF", "FastRF"]] call FUNC(gesture);};
+        _x setVariable [QGVAR(currentTask), "Manoeuvre CQB"];
+
+        systemchat "MANOEUVRE CQC!";
```

Comment:
debug message missed

Reply:
Good spot. 

Just to be clear. I intend to remove all systemchat messages not wrapped in the debug variables before release. They are only here to excessively add feedback while testing this particular set of features.